deny.toml

```diff
@@ -0,0 +1,14 @@
+[licenses]
+unlicensed = "deny"
+confidence-threshold = 1.0
+allow = [
+    "Apache-2.0",
+    "BSD-3-Clause",
+    "MIT",
+    "Unlicense"
+]
+
+exceptions = [
```

Comment:
How comes these exceptions?

Reply:
The exact status of the MPL is a bit fishy (in the sense that it's less clear than other licenses about its virality), and we really don't want to allow any more such code in unless we absolutely have to. Doing things this way means that we have to explicitly allow the MPL in each time.

Reply:
OK. So do we need to add a comment to that effect?

Reply:
Point taken. Fixed in https://github.com/softdevteam/snare/pull/30/commits/92489ab5b56daf2cd1efde0c5c437003332b745c.